main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   send_auth_rule_name   = "SendSharedAccessKey"
   listen_auth_rule_name = "ListenSharedAccessKey"
+  filter_rule_name      = "TopicFilter"
```

Comment:
I also noticed this is in the wrong repo altogether, it should be in terraform-module-servicebus-subscription